README.md

```diff
@@ -2,6 +2,14 @@
 
 The default template for a Neutralinojs app. It's possible to use your favorite frontend framework by using [these steps](https://neutralino.js.org/docs/how-to/use-a-frontend-library).
 
+## Contributing
```

Comment:
Could you please use "Contributors" as the title since we just display the contributor list? Also, we can remove the "Neutralinojs contributors:" sentence I think. Thanks